Tests/TuistGeneratorTests/Generator/LinkGeneratorTests.swift

```diff
@@ -272,6 +272,39 @@ final class LinkGeneratorErrorTests: XCTestCase {
         }
     }
 
+    func test_generateLinkingPhase_sdkNodes() throws {
+        // Given
+        let dependencies: [DependencyReference] = [
+            .sdk("/Strong/Foo.framework", .required),
+            .sdk("/Weak/Bar.framework", .optional),
+        ]
+        let pbxproj = PBXProj()
+        let pbxTarget = PBXNativeTarget(name: "Test")
+        let fileElements = ProjectFileElements()
+        let requiredFile = PBXFileReference(name: "required")
+        let optionalFile = PBXFileReference(name: "optional")
+        fileElements.sdks["/Strong/Foo.framework"] = requiredFile
+        fileElements.sdks["/Weak/Bar.framework"] = optionalFile
+
+        // When
+        try subject.generateLinkingPhase(dependencies: dependencies,
+                                         pbxTarget: pbxTarget,
+                                         pbxproj: pbxproj,
+                                         fileElements: fileElements)
+
+        // Then
+        let buildPhase = pbxTarget.buildPhases.last as? PBXFrameworksBuildPhase
```

Comment:
You can use [this getter](https://github.com/tuist/xcodeproj/blob/master/Sources/xcodeproj/Objects/Targets/PBXTarget.swift#L226)

Reply:
👍 will update - thanks!

Reply:
ahh we don't yet have the `XcodeProj` version with this change

Reply:
https://github.com/tuist/tuist/pull/412

Reply:
Merged master which has a newer version of XcodeProj and updated the test to use the helper